Documentation/users-integrations.md

```diff
@@ -0,0 +1,8 @@
+# Users and Integrations
+
+This document tracks projects, integrations, and use cases for bootkube. [Join the community](../README.md), and help us keep the list up-to-date.
+
+* [Tectonic](https://coreos.com/tectonic) - Enterprise-ready Kubernetes clusters
```

Comment:
https://github.com/poseidon/typhoon - Minimal and free Kubernetes distribution
https://github.com/poseidon/bootkube-terraform - Terraform module for rendering bootkube manifests and assets